rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -444,59 +444,21 @@ BtcBlockStoreWithCache getBtcBlockStore() {
     protected PegTxType getTransactionType(BtcTransaction btcTx) {
```

Comment:
This method should also go into PegUtilsLegacy, right? Is that part of another story?

Reply:
No really. This method is simply abstracting how `PegUtilsLegacy. getTransactionType` should be invoked. A `getTransactionType` method exists already in `PegUtilsLegacy`.